react-native-sdk/update_dependencies.js

```diff
@@ -21,6 +24,19 @@ function updateDependencies() {
             packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
             updated = true;
         }
+
+        if (semver.satisfies(RNSDKpackageJSON.peerDependencies[key], `=${packageJSON.dependencies[key]}`)) {
+            return;
+        }
+
+        if (semver.satisfies(RNSDKpackageJSON.peerDependencies[key], `>${packageJSON.dependencies[key]}`)) {
```

Comment:
Why do we need to update it if does satisfy the requirement? Isn't thjis backwards?

Reply:
No, it's not. If our peer dependencies have a bigger version than the dependencies that are in the app that is using our rnsdk, then we need to update.

Reply:
Ah right!